src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -1975,7 +1977,31 @@ private File createUnixGitSSH(File key, String user) throws IOException {
             w.println("ssh -i \"" + key.getAbsolutePath() + "\" -l \"" + user + "\" -o StrictHostKeyChecking=no \"$@\"");
         }
         ssh.setExecutable(true, true);
-        return ssh;
+        //JENKINS-48258 git client plugin occasionally fails with "text file busy" error
+        //The following creates a copy of the generated file and deletes the original
+        //In case of a failure return the original and delete the copy
+        String fromLocation = ssh.toString();
+        String toLocation = ssh_copy.toString();
+        //Copying ssh file
+	try {
```

Comment:
Since you submitted an indentation repair commit, could you also make the indentation on this line match the preceding lines?

The contributing guidelines generally prefer that white space changes be avoided so that it is easier to read diffs.  However, since you've made a change to white space, please align with the rest of the text.